### ISSUE TITLE
Suggestions and bugs

### ISSUE THREAD

**Issue by fembooooooy** [Kopamed/Raven-bPLUS#82]
- Bugs -
1. Bedwars overlay - Change [ Final kills section ] from `{final_deaths_bedwars}` to `{final_kills_bedwars}` because it displays your [ Final death ] not [ Final kills ]
2. Sumo fence - Change the fence high in `Fort Royale` map because it's down the map
3. AimAssist - Make it not fucking aim and hit the invis bots and make it less choppy 
4. Duels info - Not working. ( Or i am retarded? )
5. Bridge info - Not working. ( Or i am retarded? )
6. AntiShuffle - Doesn't show the real IGN of the Player it shows random letters

- Suggestions -
1. Make Raven B+ Screenshot friendly ( Removes the HUD or any visual enabled from the screenshot )
2. Better GUI ( WIth anime pics like Novoline idc stfu L ez bozo. )
3. Make Velocity bypass MMC please.

**Comment by Nukemo** [comment]
Hypixel updated duels so duels info dosent work
The aimassist issue is that you have not enabled antibot
Velo does bypass MMC you just need to toggle it every 15-20 hits
Just bind gui to a key to hide it then take ss

**Comment by CheaterPolska** [comment]
Add AutoArmor And Bypass AntyForge

**Comment by Krenz-Kill-Kits** [comment]
AntiForge can be bypassed by ClientNameSpoofer, turn on ClientNameSpoofer, turn on Command Line and type in "f3name Lunar Client" (w/o the quotes)

**Comment by llama-for3ver** [comment]
most of these are just a config issue

**Comment by fembooooooy** [comment]
> most of these are just a config issue

What's exactly the config issue...?

**Comment by k6d** [comment]
> AntiForge can be bypassed by ClientNameSpoofer, turn on ClientNameSpoofer, turn on Command Line and type in "f3name Lunar Client" (w/o the quotes). Or just download a Anti-Forge Bypass mod

no f3name is just a visual and doesn't change ur client name at all

**Comment by Nukemo** [comment]
yeah client name spoofer just changes.name client side only for the f3 screen use fakelunar or something to spoof client
